File: src/main/java/cib/test/cibtest/model/Sock.java
package cib.test.cibtest.model;

import lombok.Data;
import org.springframework.beans.factory.annotation.Value;

import javax.persistence.*;

@Data
@Table(name = "Socks")
@Entity
public class Sock {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;

    @Column
    private String color;

    @Column
    private Long cottonPart;

    @Column
    private Long quantity;
}

File: database/src/main/kotlin/cib/interns/test/task/database/entity/Sock.kt
package cib.interns.test.task.database.entity

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id

@Entity
class Sock {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null
}